src/test/java/com/sap/sgs/phosphor/fosstars/data/github/UsesNoHttpToolTest.java

```diff
@@ -58,14 +56,11 @@ public void testGradleWithoutNoHTTP() throws IOException {
   }
 
   private UsesNoHttpTool createProvider(InputStream is, String filename) throws IOException {
-    GHContent content = mock(GHContent.class);
-    when(content.isFile()).thenReturn(true);
-    when(content.read()).thenReturn(is);
+    final LocalRepository repository = mock(LocalRepository.class);
+    when(repository.read(filename)).thenReturn(Optional.of(is));
 
-    GHRepository repository = mock(GHRepository.class);
-    when(repository.getFileContent(filename)).thenReturn(content);
-
-    when(fetcher.github().getRepository(any())).thenReturn(repository);
+    GitHubProject project = new GitHubProject("org", "test");
+    fetcher.addForTesting(project, repository);
```

Comment:
Is there no test case to check where `LOCATION` it was expected to be found?

Reply:
Currently there is no tests for that. In fact, locations are not used at all here. They are necessary for #232. The test will be added there. However, I can also add tests for the visitors that use the existing POM files.